File: RecordMusicCunoc/src/main/java/com/adolfosc/analizadores/ErrorSintaxis.java
package com.adolfosc.analizadores;

/**
 *
 * @author hectoradolfo
 */
public class ErrorSintaxis {
    
    private String tipo;
    private String desc;
    private String lin;
    private String col;
    private String lex; 

    public ErrorSintaxis(String tipo, String desc, String lin, String col, String lex) {
        this.tipo = tipo;
        this.desc = desc;
        this.lin = lin;
        this.col = col;
        this.lex = lex;
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getLin() {
        return lin;
    }

    public void setLin(String lin) {
        this.lin = lin;
    }

    public String getCol() {
        return col;
    }

    public void setCol(String col) {
        this.col = col;
    }

    public String getLex() {
        return lex;
    }

    public void setLex(String lex) {
        this.lex = lex;
    }    
}

File: RecordMusicApp/app/src/main/java/com/adolfosc/analizadores/ErrorSintaxis.kt
package com.adolfosc.analizadores

class ErrorSintaxis {

    private var tipo: String? = null
    private var desc: String? = null
    private var lin: String? = null
    private var col: String? = null
    private var lex: String? = null

    constructor(tipo: String?, desc: String?, lin: String?, col: String?, lex: String?) {
        this.tipo = tipo
        this.desc = desc
        this.lin = lin
        this.col = col
        this.lex = lex
    }

    fun getTipo(): String? {
        return tipo
    }

    fun setTipo(tipo: String?) {
        this.tipo = tipo
    }

    fun getDesc(): String? {
        return desc
    }

    fun setDesc(desc: String?) {
        this.desc = desc
    }

    fun getLin(): String? {
        return lin
    }

    fun setLin(lin: String?) {
        this.lin = lin
    }

    fun getCol(): String? {
        return col
    }

    fun setCol(col: String?) {
        this.col = col
    }

    fun getLex(): String? {
        return lex
    }

    fun setLex(lex: String?) {
        this.lex = lex
    }
}